gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -314,7 +314,8 @@ gst_tensor_src_iio_class_init (GstTensorSrcIIOClass * klass)
       g_param_spec_string ("channels", "Channels to be enabled",
           "Enable channels -"
           "auto: enable all channels when no channels are enabled automatically"
-          "all: enable all channels",
+          "all: enable all channels"
```

Comment:
Later, please consider to show proper description using gst-inspect.
now $ gst-inspect-1.0 tensor_src_iio
```
$ gst-inspect-1.0 tensor_src_iio
..
  channels            : Enable channels -auto: enable all channels when no channels are enabled automaticallyall: enable all channels
```

Reply:
Updated.